test/runtests.jl

```diff
@@ -2,12 +2,20 @@
 using FFTW
 using Base.Test
 
-# Make super sure we don't pull in Base names
 importall FFTW
-import AbstractFFTs: Plan, fft, ifft, bfft, fft!, ifft!, bfft!,
+if VERSION >= v"0.7.0-DEV.602"
+    import AbstractFFTs: Plan, fft, ifft, bfft, fft!, ifft!, bfft!,
```

Comment:
It's the same thing we did with Primes and continue to do with SpecialFunctions, QuadGK, the priority queue stuff in DataStructures, etc. It's certainly not ideal. Being able to do that would make it easier to fully support multiple versions at once.